src/common/ChatterSet.cpp

```diff
@@ -62,6 +62,22 @@ std::vector<QString> ChatterSet::filterByPrefix(const QString &prefix) const
     return result;
 }
 
+std::vector<std::pair<QString, QString>> ChatterSet::allExcept(
+    const QString &userName) const
+{
+    std::vector<std::pair<QString, QString>> result;
+
+    for (auto &&item : this->items)
```

Comment:
```suggestion
    for (const auto &item : this->items)
```

We can't move out of the item, since the method is `const` (I know the other methods do the same).